crates/library/std/src/evm.fe

```diff
@@ -238,40 +238,40 @@ pub unsafe fn call_data_size() -> u256 {
     return __calldatasize()
 }
 
-pub fn call_data_copy(buf: MemoryBuffer, from_offset f: u256) {
-   unsafe { __calldatacopy(buf.offset(), f, buf.len()) }
+pub fn call_data_copy(mut buf: MemoryBuffer, from_offset f: u256) {
```

Comment:
There are more functions that accept immutable `buf`s (e.g `call_data_copy`, `log` , `return_mem`), do we need to change those also  ?

Reply:
`call_data_copy`?? typo?

the other ones are not writing to the buf, so they can stay as immutable